### DISCUSSION TITLE
about the symbol downloading...

### DISCUSSION THREAD

**Discussion by biliLiveTiles** [Maplespe/DWMBlurGlass#223] [Q&A]
Download failed! Unable to download file from "msdl.microsoft.com".
ps: i'm already connected to an usa vpn

**Comment by ALTaleX531** [comment]
你好，请确认你使用的版本是支持的Windows预览版，以及不是ARM系统，最后检查一下自己本身能不能访问msdl.microsoft.com，如果不能访问建议换个梯子

**Comment by Zenguycolorado** [comment]
This answer makes no sense. The symbols cannot be downloaded in Windows (not ARM) 10 and for some, Windows 11 as well.

**Comment by bbmaster123** [comment]
This happens every time a new build is released. Symbols have to become available before we can download them, which is why there's always that delay
Might be a good idea to add a line to the readme or the installer output window, just to cut down on people making new posts about this after every update

**Comment by Zenguycolorado** [comment]
Thank you for this Bbmaster123. This does make sense. Though I haven't had it take so long before, hence my worrying that something was actually broken. Things are now indeed working properly once more. Thank you again.
